module/library/individualAssessment/validator/v1.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * name : v1.js
+ * author : Aman
+ * created-date : 04-Jun-2020
+ * Description : Institutional solution validation.
+ */
+
+module.exports = (req) => {
+
+    let templateValidator = {
+
+        details : function () {
+            req.checkParams('_id').exists().withMessage("required observation template solution id");
```

Comment:
@aman-tunerlabs - Why is a validation message containing the term "observation"?